public/js/main.js

```diff
@@ -101,3 +85,42 @@ function renderToolbox() {
     document.querySelector("#mount")
   );
 }
+
+if(process.env.NODE_ENV !== "DEVTOOLS_PANEL") {
```

Comment:
I'd like to see the feature / environment checking move to `feature.js`. The rationale is that if all of the configuration were in config files, then it would be clear where to look.

This expression could be `isDevtoolsEnvironment`

Here is some pseudo-code for feature.js
https://gist.github.com/6bf0262dd14ae0d0abe0b291d198e3be